src/main/scala/org/kframework/minikore/Scanner.scala

```diff
@@ -49,7 +49,7 @@ class Scanner {
           input.next()
         } else if (!yieldEOL) { // end of line
           yieldEOL = true
-          '\n'
+          ' '   // safer than '\n' (platform independent)
```

Comment:
I can understand why you did this, but it could be misleading for the new members. I have no better idea though. Maybe, more detailed comments may help..